src/main.rs

```diff
@@ -16,6 +16,8 @@ mod grid;
 mod cell;
 mod seeds;
 
+const UPDATES_PER_SECOND: u32 = 30;
```

Comment:
You could add a command line flag for this, if you wanted. I went for 30 updates per second because it seemed to be fast enough, without being too fast to see.